src/Sylius/Bundle/ApiBundle/Resources/config/services/command_handlers.xml

```diff
@@ -101,5 +101,17 @@
             <argument type="service" id="sylius.order_processing.order_processor" />
             <tag name="messenger.message_handler" bus="sylius_default.bus"/>
         </service>
+
+        <service id="Sylius\Bundle\ApiBundle\CommandHandler\SendOrderConfirmationHandler">
+            <argument type="service" id="sylius.email_sender" />
+            <argument type="service" id="sylius.repository.order" />
+            <tag name="messenger.message_handler" bus="sylius_default.bus" />
+        </service>
+
+        <service id="Sylius\Bundle\ApiBundle\CommandHandler\OrderCompletedHandler">
```

Comment:
If I see it correctly, it is not a command handler, but sth like event handler

Reply:
I moved it to event_listeners.xml